### PR TITLE
[IBCDPE-88] philosopy

### LINKED ISSUE #2: Finish philosophy section
The philosophy section is only partially stubbed

### PR DESCRIPTION
**Pull request by tthyer**
proposed philosophy -- needs ~~criticism~~ feedback

depends on #3

### CODE REVIEW

**Comment by thomasyu888** [comment]
I like it - this is less of feedback then just a slight re-write. Let me know your thoughts.


> Coding conventions are a MUST when we want successful collaboration.  We are plural, and we want to work together on these conventions in a deliberate way.  This is an evolving document - we want to repeatedly discuss, write, and review these guidelines.

> For coders, the code is part of how we live - it’s similar to how you organize your residence. To some people, it really matters where the milk is in the refrigerator, or whether the utensils are put in the dishwasher pointy-side up or down. These are the conventions and the preferences.  But, other things are best practices -- you really SHOULD change those batteries in your fire alarm once per year. Still others are REQUIRED, like having to pay property taxes or rent every year.

> Coding conventions allow us to get our work done smoothly in an optimally social way.

> This is a handbook of both these conventions as well as practical tips on how to get things done.

**Comment by tthyer** [comment]
@thomasyu888 Re-read this a few times and decided I really like your rewrite except not so much the very first line. That said, I wanted to hold off on this PR until the code space meeting on Friday to get more feedback from the group. cc @mattfazza

**Comment by mattfazza** [comment]
### 1st Paragraph: small rewrite.

> _Coding conventions are pivotal in successful collaboration._ We are plural, and we want to work together on these conventions in a deliberate way.  This is meant to be an evolving document - we want to repeatedly discuss, write, and review these.

### 2nd Paragraph: if this repository is intended for an audience who codes, I don't see this entire comparison being necessary. 

> For coders, the code is part of how we live - it’s similar to how you organize your residence. To some people, it really matters where the milk is in the refrigerator, or whether the utensils are put in the dishwasher pointy-side up or down. These are the conventions and the OPTIONAL but RECOMMENDED preferences. But, other things are best practices -- you really SHOULD change those batteries in your fire alarm once per year. Then there are the REQUIRED, like having to pay property taxes or rent every year.

### 3rd Paragraph: small addition.

> Coding conventions allow us to get our work done smoothly in an optimally social way, _by adding predictability, uniformity, and proven standards to the way we write code._

### 4th Paragraph: small revision.

> This handbook documents these conventions as well as practical tips on how to get things done.

**Review by mfazza — CHANGES_REQUESTED** [review]
It's in the comments.  Let's discuss

**Comment by tthyer** [comment]
@thomasyu888 @mattfazza Thanks! I'm gathering together the content suggestions. As for specific word choices and phrasing, I'm going to be meeting with Zoe / other writing experts.

**Comment by thomasyu888** [comment]
Thanks Matt, Tess for drafting and reviewing.  I discussed with Zoe and this was the text we came up with.  I will make modifications to the PR soon.

> We create and maintain coding conventions to ensure successful collaboration. We want to work together on these conventions in a deliberate way, repeatedly discussing, writing, and reviewing this evolving document.

> For coders, the code is part of how we live - it’s similar to how you organize your residence. To some people, it really matters that forks face handle-up in the dishwasher, or that the toilet paper hangs over the roll. These are the conventions and the optional but recommended preferences. But, other things are best practices -- you really should change those batteries in your fire alarm once per year. Then there are the required, like having to pay utilities.

> Coding conventions add predictability, uniformity, and proven standards to the way we write code, which allows us to work more efficiently. This handbook documents these conventions and suggests practical tips.

File: content/pages/000_meta/philosophy.md [hunk 1]
```diff
@@ -5,8 +5,8 @@ Slug: philosophy
 Summary: Philosopy
 Status: published
 
-* needs work*
+We create and maintain coding conventions to ensure successful collaboration. We want to work together on these conventions in a deliberate way, repeatedly discussing, writing, and reviewing this evolving document.
```
Comment (mattfazza) [inline content/pages/000_meta/philosophy.md:9]: I like the way this is written.

File: content/pages/000_meta/philosophy.md [hunk 2]
```diff
@@ -5,8 +5,8 @@ Slug: philosophy
...
 
-*TODO: we should explain why we need coding conventions. Why are we bothering about this? I keep thinking in metaphors like, for coders this is how we live. It is like what you prefer to do in your own house. It really matters to some people where you put the milk in the refrigerator, or whether the utensils are put in the dishwasher pointy-side up or down.*
+For coders, the code is part of how we live - it’s similar to how you organize your residence. To some people, it really matters that forks face handle-up in the dishwasher, or that the toilet paper hangs over the roll. These are the conventions and the optional but recommended preferences. But, other things are best practices -- you really should change those batteries in your fire alarm once per year. Then there are the required, like having to pay utilities.
 
-*Alternately, some of the conventions are ways to make your work go more smoothly, or are forms of etiquette that help in collaboration.*
+Coding conventions add predictability, uniformity, and proven standards to the way we write code, which allows us to work more efficiently. This handbook documents these conventions and suggests practical tips.
```
Comment (mattfazza) [inline content/pages/000_meta/philosophy.md:12]: +1 for this one. 🥇

File: content/pages/000_meta/philosophy.md [hunk 3]
```diff
@@ -5,8 +5,8 @@ Slug: philosophy
 Summary: Philosopy
 Status: published
 
-* needs work*
+We create and maintain coding conventions to ensure successful collaboration. We want to work together on these conventions in a deliberate way, repeatedly discussing, writing, and reviewing this evolving document.
 
-*TODO: we should explain why we need coding conventions. Why are we bothering about this? I keep thinking in metaphors like, for coders this is how we live. It is like what you prefer to do in your own house. It really matters to some people where you put the milk in the refrigerator, or whether the utensils are put in the dishwasher pointy-side up or down.*
+For coders, the code is part of how we live - it’s similar to how you organize your residence. To some people, it really matters that forks face handle-up in the dishwasher, or that the toilet paper hangs over the roll. These are the conventions and the optional but recommended preferences. But, other things are best practices -- you really should change those batteries in your fire alarm once per year. Then there are the required, like having to pay utilities.
```
Comment (mattfazza) [inline content/pages/000_meta/philosophy.md:11]: I'm not against this part, I just don't find it necessary.

**Review by mfazza — APPROVED** [review]

**Pull request merged**